src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_logging.v2beta1.json.baseline

```diff
@@ -24,24 +32,29 @@ google.auth.getApplicationDefault(function(err, authClient) {
   }
 
   var request = {
-    // TODO: Change placeholders below to appropriate parameter values for the 'list' method:
 
-    resource: {},
+    resource: {
+      // TODO: Add the desired properties.
+    },
 
-    // Auth client
     auth: authClient
   };
 
-
-  var recur = function(err, result) {
+  var recur = function(err, response) {
```

Comment:
This is not a part of the main patch of this, but this style of `var recur = function(...)` does not look quite JS-ish style to me. Instead it would be `function recur(err, response) { ...`, or simply embedding it into the invocation as
```javascript
logging.entries.list(request, function(err, response) {
   ...
});
```
is more idiomatic.